runtime/oti/VMHelpers.hpp

```diff
@@ -2086,7 +2151,7 @@ class VM_VMHelpers
 #if JAVA_SPEC_VERSION >= 19
 		jboolean started = J9VMJDKINTERNALVMCONTINUATION_STARTED(vmThread, continuationObject);
 		J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, continuationObject);
-		needScan = started && (NULL != continuation) && (!scanOnlyUnmounted || !isContinuationMounted(continuation));
+		needScan = started && (NULL != continuation) && (!scanOnlyUnmounted || !isContinuationMountedOrConcurrentlyScanned(continuation));
```

Comment:
Do we need scanOnlyUnmounted  anymore - Is there a case left that we pass it as false?

Reply:
will remove